website/pages/docs/agent/options.mdx

```diff
@@ -1146,6 +1146,12 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
     The default value is "No limit" and should be tuned on large
     clusters to avoid performing too many RPCs on entries changing a lot.
 
+  - `use_streaming_backend` When enabled Consul client agents will use streaming rpc to
+    populate the cache, instead of the traditional blocking queries. All servers must
+    have `rpc.enable_streaming` enabled before any client can enable `use_streaming_backend`.
```

Comment:
Looking at the docs, I guess these should be links to the other settings, to make it easier to jump to them.